R/plumber.R

```diff
@@ -242,7 +248,9 @@ Plumber <- R6Class(
         path <- paste0(path, "/")
       }
 
+      # order the mounts so that the more specific mount paths are ahead of the less specific mount paths
```

Comment:
```suggestion
      # Add the mount to the original mount order
      private$mnt_order <- c(private$mnt_order, path)
      
      # Order the mounts so that the more specific mount paths are ahead of the less specific mount paths
```